content/management/configuration-api/v3.2/index.mdx

```diff
@@ -32,6 +32,239 @@ Summing up, if the user token starts with `fra-`, you should add the `X-Region:
 
 All configurations set by this API will have action in system after max 2 minutes. This delay will be removed in the future.
 
+# Auto access
+
+**Auto access** allows you to assign groups to a thread that will be created when a customer starts a chat. Those groups are
+assigned based on the conditions defined to match the customer's tracking URL and geolocation. Every time a customer calls
```

Comment:
`tracking URL` sounds difficult, maybe something like the current customer's page URL?

Reply:
I don't know.  @ale-kacperczyk , wdyt?

Reply:
Hm, it may sound a bit enigmatic. 
On the other hand, `current customer's page URL` sounds a bit as if the customer had his/her own website.
I suggest "...to match the customer's geolocation and the page URL the customer is currently visiting. Every time... "